src/cluster.c

```diff
@@ -5532,6 +5534,12 @@ NULL
             (unsigned long long) myepoch
         );
 
+        if (nodeIsSlave(myself) && myself->slaveof) {
+            info = sdscatprintf(info,
+                                "cluster_my_replica_epoch:%llu\r\n",
+                                (unsigned long long) myself->configEpoch);
```

Comment:
I think we need a better name here if we want to expose this. It sounds like this is the node's replica's epoch, as compared to, "If I am a replica, show my epoch". I'm not really all that convinced this is needed to be shown.

Reply:
I thought that it would be best to rename 'cluster_my_epoch' on replicas to 'cluster_my_primary_epoch', and have another new entry of 'cluster_my_epoch' which will point to their real config epoch.  However, that breaks backward compatibility for anyone using this entry in a weird way to identify nodes in the same shard or something like this. 
We can also change it to cluster_replica_epoch / cluster_replica_real_epoch / cluster_replica_my_epoch.
WDYT? do you have other suggestions?

IMHO we should make it available as this info can be useful for debugging failovers and slot migrations.

Reply:
I honestly "don't" think we should be really exposing epoch here. This is mostly internal state, which is actually very hard for the vast majority of end users to parse and understand.

Reply:
Ok, I can remove this part. What do you think about the rest of the change?

Reply:
I think the rest makes sense.

Reply:
Removed the replica's epoch